AppCenterReactNativeShared/zip-framework.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 cd Products
-wrapperSdkString=$(grep s.version AppCenterReactNativeShared.podspec)
+podspecFile="${podspecFile:-AppCenterReactNativeShared.podspec}"
```

Comment:
Can you explain this line?

Reply:
Will add a comment.

`podspecFile` is parameterized but its default value is `podspecFile:-AppCenterReactNativeShared.podspec` when it's not set.